scripts/win32.bat

```diff
@@ -54,5 +56,14 @@ For Each objDrive In colDiskDrives
             Wscript.Echo ""
         Next
     Next
+    If foundLDs = 0 Then
+      Wscript.Echo "device: """ & Replace(objDrive.DeviceID, "\", "\\") & """"
+      Wscript.Echo "description: """ & objDrive.Caption & """"
+      Wscript.Echo "size: " & objDrive.Size
+      Wscript.Echo "mountpoint: Null"
+      Wscript.Echo "name: Null"
+      Wscript.Echo "system: False"
```

Comment:
I believe we should default to `system: True` here, or we should find a way to better detect this. As it is now, a program using this module, might detect any non-mountable disk drive as a removable one.